### PR TITLE
Remove unsafe `atty` dependency

### PR DESCRIPTION
**Pull request by Kerollmops**
This PR replaces the `atty` dependency with the `is-terminal` one. We do that to fix GHSA-g98v-hv3f-hcfr.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -36,24 +36,3 @@ opt-level = 3
...
-opt-level = 3
-[profile.release.package.yada]
-opt-level = 3
-
-[profile.bench.package.lindera-ipadic-builder]
-opt-level = 3
-[profile.bench.package.encoding]
-opt-level = 3
-[profile.bench.package.yada]
-opt-level = 3
```
Comment (dureuill) [inline Cargo.toml:59]: why are we removing these?

**Review by dureuill — COMMENTED** [review]
looks good thank you, just a small comment :-)

File: Cargo.toml (hunk 1 above)
Comment (Kerollmops) [reply Cargo.toml:59]: Wow! I didn't remember doing that by hand. It looks like `cargo add is-terminal` or `cargo remove atty` did that, thank you @dureuill !

Comment (Kerollmops) [reply Cargo.toml]: Fixed !

**Review by dureuill — APPROVED** [review]
Thanks a lot kero :+1:

**Comment by dureuill** [comment]
Bors merge

**Comment by dureuill** [comment]
bors merge

**Pull request merged**